CommonComReader.py

```diff
@@ -16,14 +16,52 @@
 from UM.PluginRegistry import PluginRegistry
 from UM.Scene.SceneNode import SceneNode
 
-# PyWin32
-import comtypes
-from comtypes.client import GetClassObject
+# Trying to import one of the COM modules
+try:
 
+    import win32com.client
+    import win32com.client.gencache
+    import pythoncom
+    import pywintypes
+
+    class ComFactory:
+        def CreateClassObject(app_name):
+            win32com.client.gencache.Rebuild()
+            #win32com.client.gencache.EnsureModule(app_name)
+            return win32com.client.gencache.EnsureDispatch(app_name)
+            res = win32com.client.Dispatch("{F16137AD-8EE8-4D2A-8CAC-DFF5D1F67522}")
+            Logger.log("d", "DIR> %s", dir(res()))
+            return res
+            #return win32com.client.Dispatch(app_name)
+
+        def Coinit():
+            pythoncom.CoInitializeEx(pythoncom.COINIT_MULTITHREADED)
+
+        def UnCoinit():
+            pythoncom.CoUninitialize()
+
+    Logger.log("i", "ComFactory: Using pywintypes!")
+
+except ImportError:
+    ## ComTypes
+    import comtypes
+    import comtypes.client
+
+    class ComFactory:
+        def CreateClassObject(app_name):
+            return comtypes.client.GetClassObject(app_name).CreateInstance()
+
+        def Coinit():
```

Comment:
Shouldn't this be coInitialize? (eg; co & init are different words right?)

Reply:
Good point. Will fix that..